build-aux/Jenkinsfile.pr

```diff
@@ -247,7 +247,7 @@ pipeline {
         axes {
           axis {
             name 'ERLANG_VERSION'
-            values '25.3.2.13', '26.2.5.2', '27.0.1'
+            values '25.3.2.13', '26.2.5.4', '27.1.1'
```

Comment:
there should be 25.3.2.14 as well …

Reply:
Nah, didn't build 25.3.2.14. Not enough fixes in it to justify burning all the CPUs on my laptop